salt/metalk8s/map.jinja

```diff
@@ -56,111 +95,163 @@
   'default': {}
 }, merge=defaults.get('kubeadm_preflight')) %}
 
-{% set repo = salt['grains.filter_by']({
-  'RedHat': {
-    'containerd': {
-      'name': 'epel'
-    },
-    'kubernetes': {
-      'name': 'kubernetes'
-    },
-    'repositories': {
-      'metalk8s-epel': {
-        'humanname': 'EPEL',
-        'gpgcheck': 1,
-        'gpgkeys': [
-            'RPM-GPG-KEY-metalk8s-epel-7_1',
-        ],
-        'repo_gpg_check': 0,
-        'enabled': 1,
-      },
-      'metalk8s-kubernetes': {
-        'humanname': 'Kubernetes',
-        'gpgcheck': 1,
-        'gpgkeys': [
-            'RPM-GPG-KEY-metalk8s-kubernetes-7_1',
-            'RPM-GPG-KEY-metalk8s-kubernetes-7_2',
-        ],
-        'repo_gpg_check': 0,
-        'enabled': 1,
-      },
-      'metalk8s-saltstack': {
-        'humanname': 'SaltStack',
-        'gpgcheck': 1,
-        'gpgkeys': [
-            'RPM-GPG-KEY-metalk8s-saltstack-7_1',
-        ],
-        'repo_gpg_check': 0,
-        'enabled': 1,
-      },
-      'metalk8s-scality': {
-        'humanname': 'Scality',
-        'gpgcheck': 0,
-        'gpgkeys': [],
-        'repo_gpg_check': 0,
-        'enabled': 1,
-      },
-    }
-  },
-  'Debian': {
-    'repositories': {
-      'metalk8s-bionic': {
-        'type': 'deb',
-        'options': {
-          'trusted': 'yes',
+{% set repo = salt['grains.filter_by'](
+  salt['grains.filter_by']({
+    'RedHat': {
+      '7': {
+        'containerd': {
+          'name': 'epel'
```

Comment:
`containerd` is no longer the one from `epel`.

Reply:
Yes, didn't change anything, it was already there, but those entries (`repo.containerd` and `repo.kubernetes`) are leftovers and should be cleaned up. I'll remove them.